### DISCUSSION TITLE
How to use the package?

### DISCUSSION THREAD

**Discussion by Marchiuzzz** [spatie/laravel-database-mail-templates#45] [Q&A] [answered]
I don't understand if I'm that stupid or the usage guide is incomplete but everyone else manages to figure out how to use it. In the example seeder you're using 
```
MailTemplate::create([
            'mailable' => \App\Mails\WelcomeMail::class,
            'subject' => 'Welcome, {{ name }}',
            'html_template' => '<h1>Hello, {{ name }}!</h1>',
            'text_template' => 'Hello, {{ name }}!',
        ]);
```
How would my app know about MailTemplate if I don't import it? I've tried it and I do get an error `Class 'App\\MailTemplate' not found` I've tried adding `use Spatie\MailTemplates\MailTemplate;` at the top but that also didn't work - got `Class 'Spatie\MailTemplates\MailTemplate' not found`

How do you use the `MailTemplate` class?

**Comment by Marchiuzzz** [comment, marked as answer]
Well I found the solution - need to use `Spatie\MailTemplates\Models\MailTemplate` instead of `Spatie\MailTemplates\MailTemplate`. I suppose the documentation is a bit outdated.

**Reply by freekmurze** [reply]
I've updated the docs. If you see other stuff that can be correct, do send a PR 👍
